compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/Compiler.java

```diff
@@ -74,7 +80,10 @@ public BLangPackage compile(String sourcePackage) {
     }
 
     public void build() {
-        compilePackages().forEach(this.binaryFileWriter::write);
+        List<BLangPackage> packageList = compilePackages();
+        packageList.forEach(this.binaryFileWriter::write);
+        packageList.forEach(bLangPackage -> lockFileWriter.getEntryPkgDependencies(bLangPackage.symbol));
```

Comment:
`getEntryPkgDependencies` should be `addEntryPkg`,  isn't it?

Reply:
Changed it to addEntryPkg with 7340531